<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomProfileDialog</class>
 <widget class="QDialog" name="CustomProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Custom Video Mode</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="7" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Colorspace</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>colorspaceCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="colorspaceCombo">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string notr="true">ITU-R BT.601</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">ITU-R BT.709</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Resolution</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>widthSpinner</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QSpinBox" name="aspectNumSpinner">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="aspectDenSpinner">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="value">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="aspectRatioComboBox">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string notr="true">16:9 (wide)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">4:3 (SD)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">9:16 (vertical)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">1:1 (square)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">1.90:1 (DCI)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>nameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QComboBox" name="scanModeCombo">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Interlaced</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Progressive</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Aspect ratio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>aspectNumSpinner</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QSpinBox" name="widthSpinner">
          <property name="minimum">
           <number>16</number>
          </property>
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1280</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="heightSpinner">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>720</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resolutionComboBox">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string notr="true">1280 x 720 (HD)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">1920 x 1080 (FHD)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">2048 x 1080 (DCI 2K)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">2560 x 1440 (QHD 2K)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">2704 x 1520 (2.7K)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">3840 x 2160 (UHD 4K)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">4096 x 2160 (DCI 4K)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">5120 x 2880 (5K)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Frames/sec</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>fpsSpinner</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QDoubleSpinBox" name="fpsSpinner">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fpsComboBox">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string notr="true">23.976024</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">29.970030</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">48</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">59.940060</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">60</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Scan mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>scanModeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;small&gt;(Leave Name blank to skip saving a preset and use a temporary or project-specific Video Mode.)&lt;/small&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>widthSpinner</tabstop>
  <tabstop>heightSpinner</tabstop>
  <tabstop>aspectNumSpinner</tabstop>
  <tabstop>aspectDenSpinner</tabstop>
  <tabstop>fpsSpinner</tabstop>
  <tabstop>scanModeCombo</tabstop>
  <tabstop>colorspaceCombo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
